src/data/reducers/siteInfo.js

```diff
@@ -30,6 +31,7 @@ const siteInfo = (state = {
         visitedPages: action.siteInfo.visited_pages,
         themeName: action.siteInfo.theme_name,
         logo: action.siteInfo.logo,
+        footerLinks: action.siteInfo.footer_links,
```

Comment:
This will probably be `footerLinks: action.siteInfo.footer_links.links` with `links` being whatever you name the parent key in the JSON object in edx/journals#125 (based on JSON rules commented)